### PR TITLE
Adding value props to docs

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _general/integrations/codeclimate.md [hunk 1]
```diff
@@ -31,7 +31,11 @@ redirect_from:
 
 ## About Code Climate
 
-Code Climate is an automated code coverage service. Starting with Code Climate and Codeship is fast and easy. [Their documentation](http://docs.codeclimate.com/article/219-setting-up-test-coverage) does a great job of providing more information, in addition to the setup instructions below.
+Code Climate is an automated code coverage service. Starting with Code Climate and Codeship is fast and easy.
+
+By using Code Climate you can help enforce higher standards of code quality and transparency with your engineering tea.
```
Comment (manualwise) [inline _general/integrations/codeclimate.md]: "with your engineering tea." <-- M is missing

File: _general/integrations/codecov.md [hunk 2]
```diff
@@ -21,7 +21,11 @@ menus:
 
 ## About Codecov
 
-Codecov is an automated code coverage service. Starting with Codecov and Codeship is fast and easy. [Their documentation](https://docs.codecov.io/docs/) does a great job of providing more information, in addition to the setup instructions below.
+Codecov is an automated code coverage service. Starting with Codecov and Codeship is fast and easy.
+
+By using Codecov you can help enforce higher standards of code quality and transparency with your engineering tea.
```
Comment (manualwise) [inline _general/integrations/codecov.md]: same as on codeclimate --> "teaM".

File: _general/integrations/coveralls.md [hunk 3]
```diff
@@ -26,7 +26,11 @@ redirect_from:
 
 ## About Coveralls
 
-Coveralls is an automated code coverage service. Starting with Coveralls and Codeship is fast and easy. [Their documentation](https://coveralls.zendesk.com/hc/en-us/categories/200131159-Documentation) does a great job of providing more information, in addition to the setup instructions below.
+Coveralls is an automated code coverage service. Starting with Coveralls and Codeship is fast and easy.
+
+By using Coveralls you can help enforce higher standards of code quality and transparency with your engineering tea.
```
Comment (manualwise) [inline _general/integrations/coveralls.md]: "team"

**Comment by mlocher** [comment]
Thanks for catching those, @manualwise :bow:

**Pull request merged**